apache/superset
# chore: Feature flag for embedding charts

### SUMMARY
Users of Superset might not want to give the ability to embed charts, especially as embedding uses iFrames that might be disabled for security reasons. This PR adds a `EMBEDDABLE_CHARTS` flag to the config to enable/disable embedding charts. Default to `True`.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

<img width="1173" alt="Screenshot 2022-07-21 at 15 58 51" src="https://user-images.githubusercontent.com/60598000/180219382-d24904d4-0fdf-411f-b22c-f731470a2c1c.png">

<img width="1181" alt="Screenshot 2022-07-21 at 15 59 29" src="https://user-images.githubusercontent.com/60598000/180219412-02ce3212-44cd-4632-939f-21c2ff03e814.png">

### TESTING INSTRUCTIONS
Set the feature flag off and make sure the Embed code option isn't appearing as a Share option in Explore.

### ADDITIONAL INFORMATION
